### DISCUSSION TITLE
SAM picker doesnt work

### DISCUSSION THREAD

**Discussion by XsaidyX** [gibbed/SteamAchievementManager#406] [Q&A] [answered]
Hi, I have encountered such a problem, recently SAM PICKER has been working unstable, or does not work at all, when starting a game in which I want to hack achievements, I get the error "scheme loading error", I disassembled a lot, reinstalled, launched as administrator, but it does not help, tell me what To do? 
![image](https://github.com/user-attachments/assets/8fc11310-67ba-48c2-b139-7486524f1f38)

**Comment by DeuceINC01** [comment]
If you have copied the app number, then click Add Game in the box. Make sure you enter the correct app number before opening it.

**Comment by qnthxny** [comment]
I'm having the same problem do you know how to fix it

**Comment by BrotherClotten** [comment]
Hi, so here is the way to get SAM to work in 2024 as of September 4th.

With Steam and SAM running, launch the game you wish to unlock the achievements for in STEAM FIRST.
When you get to the main title screen of the game windows tab back to your desktop and launch the relevant SAM game.
(eg 
1: Launch CSGO in steam and wait to get to the main screen
2: windows key to desktop
3: open SAM CSGO
4: once SAM page has populated the achieve info close the STEAM game)

All achieves and info for the game will populate the SAM page.
Basically you are forcing SAM to "see" the game details. You are then free to quit the steam game but leave the SAM version running.
This procedure has to be done every time you start Steam.

**Comment by levelingCt** [comment]
I have the same problem, does anyone know how to make it work again?

**Comment by gibbed** [comment, marked as answer]
Should be addressed as of b3a66df0d1a870e1646a259698a670f6884fa7c9.
